src/transformers/models/hubert/configuration_hubert.py

```diff
@@ -149,7 +151,7 @@ def __init__(
         hidden_dropout=0.1,
         activation_dropout=0.1,
         attention_dropout=0.1,
-        feat_proj_dropout=0.1,
+        feat_proj_dropout=0.0,
```

Comment:
this was a wrong default - the `feat_proj_dropout` is pretty much never used in fairseq